server/utils/ssr.tsx

```diff
@@ -121,6 +130,11 @@ export const generateMetaComponents = (metaProps: MetaProps) => {
 				<meta key="c5" name="citation_inbook_title" content={collection.title} />,
 				<meta key="c6" name="citation_book_title" content={collection.title} />,
 				<meta key="c7" name="citation_isbn" content={collection.metadata?.isbn} />,
+				<meta
+					key="c8"
+					name="citation_publisher"
+					content={communityPublisher || 'PubPub'}
+				/>,
```

Comment:
sorry, I think I must have gotten this wrong. This should apply to all types of collections.

Reply:
the meta tag for publisher should be set on every collection, but publisher should only appear in exports for books and conferences in the exports and crossrefs?

Reply:
I know how crazy that sounds. But that is correct. The HTML metadata gets read by Google Scholar and Zotero and a few others, who do care about publisher for journals.

But crossref's schema doesn't have a publisher for journals, apparently, so here we are.